OSDOCS-16409: Cluster Admin CQAs (#116629)
openshift/openshift-docs: modules/rosa-enabling-autoscaling-nodes-about.adoc

// Module included in the following assemblies:
//
// * rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.adoc

:_mod-docs-content-type: CONCEPT
[id="nodes-enabling-autoscaling-nodes_{context}"]
= Enabling autoscaling nodes on a cluster

[role="_abstract"]

> You can enable autoscaling on worker nodes to increase or decrease the number of nodes available by editing the machine pool definition for an existing cluster.

Reviewer: ```suggestion
Disable autoscaling on worker nodes to set a fixed number of nodes available by editing the machine pool definition for an existing cluster.
```
Was suggesting this change in some comment above.
